website/util/akismet.py

```diff
@@ -66,11 +69,15 @@ def check_comment(self, user_ip, user_agent, **kwargs):
         data['user_ip'] = user_ip
         data['user_agent'] = user_agent
 
-        res = requests.post(
-            '{}{}.{}/1.1/comment-check'.format(self.API_PROTOCOL, self.apikey, self.API_HOST),
-            data=data,
-            headers=self._default_headers
-        )
+        try:
+            res = requests.post(
+                '{}{}.{}/1.1/comment-check'.format(self.API_PROTOCOL, self.apikey, self.API_HOST),
+                data=data,
+                headers=self._default_headers,
+                timeout=5
+            )
+        except RequestException as e:
+            raise AkismetClientError(reason=e.message)
```

Comment:
`message` doesn't appear to be an attribute of `RequestException` objects. source: https://github.com/kennethreitz/requests/blob/master/requests/exceptions.py#L12 . 

I think you will need something like:

```python
e.response.text
```